### PR TITLE
Add security token note to the setup in the README

### PR DESCRIPTION
**Pull request by filiptronicek**
⬆️⬆️⬆️

### CODE REVIEW

**Comment by filiptronicek** [comment]
@jeanp413 maybe I should add a screenshot? Is that too much for such a simple thing?

File: README.md [hunk 1]
```diff
@@ -1,75 +1,92 @@
...
-* [File an issue](https://github.com/microsoft/vscode/issues)
-* Follow [@code](https://twitter.com/code) and let us know what you think!
+- [Download the latest release](https://github.com/gitpod-io/openvscode-server/releases/latest)
+- Untar and run the server:
+```bash
+tar -xzf openvscode-server-v${OPENVSCODE_SERVER_VERSION}.tar.gz
+cd openvscode-server-v${OPENVSCODE_SERVER_VERSION}
+./server.sh
+```
+- Visit [localhost:3000](http://localhost:3000).
```
Comment (akosyakov) [inline README.md:55]: It should be changed too.

Comment (filiptronicek) [reply README.md:55]: Thanks,  didn't catch that one.

**Review by jeanp413 — APPROVED** [review]
LGTM, I don't think adding a screenshot is necessary

**Comment by vq0599** [comment]
I find the nightly image needs an access token but the latest image does not.
Does something change? Could you tell me the relevant commit?

**Comment by jeanp413** [comment]
@vq0599 we just did a new stable release today so now nightly and latest images require the access token

**Comment by shagr4th** [comment]
Any way to disable this ? The docker container emits a new token at each restart, it's quite overkill to read logs and paste the new URL at each image change (especially with multiple client devices)

**Comment by jeanp413** [comment]
yeah that's right `--connectionToken` will set a fixed token, there's also `--connection-secret` where you can pass the path to a file with the secret
@filiptronicek @akosyakov I think it would be a good idea to add this options to the readme

**Pull request merged**